app/models/claim.rb

```diff
@@ -45,6 +44,7 @@ class Claim < ApplicationRecord
   enum student_loan_plan: STUDENT_LOAN_PLAN_OPTIONS
 
   belongs_to :eligibility, polymorphic: true
+  belongs_to :check, optional: true
   accepts_nested_attributes_for :eligibility, update_only: true
```

Comment:
Can we flip these? It helps cognitively (for me at least), to keep the nested attribute stuff next to the relationship.